.github/workflows/release.yml

```diff
@@ -106,6 +118,9 @@ jobs:
           docker exec -t k-package-docker-test-jammy-${GITHUB_SHA} bash -c 'cd ~ && kompile test.k --backend llvm'
           docker exec -t k-package-docker-test-jammy-${GITHUB_SHA} bash -c 'cd ~ && kompile test.k --backend haskell'
           docker image push ${DOCKERHUB_REPO}:${version_tag}
+          docker cp ~/.m2/settings.xml k-package-docker-test-jammy-${GITHUB_SHA}:~/.m2/settings.xml
```

Comment:
I don't understand tihs `docker cp ...` command? Are we copying a file from the host into the docker image? Why? Shouldn't everything happen inside the docker container?

Reply:
Even more confusing, why is this coming from the host machines home directory, rather than some specific file in teh K repo?